site/.vitepress/cache/deps/_metadata.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
we should .gitignore cache, template here https://github.com/vuejs/vitepress/blob/main/.gitignore